From 86a9bb75031518543f68acae2af3a29d89c7bdd6 Mon Sep 17 00:00:00 2001
From: Davor Joja <davorjoja@logicbricks.com>
Date: Mon, 23 Jun 2014 19:58:13 +0200
Subject: [PATCH 0392/1566] gpu: drm: xylon: Moved control and interrupt
 functions to hw header file

Moved control and interrupt function declarations to hw header file.
This keeps mentioned functionality usable only from crtc driver who is
responsible for it.
Updated define align spaces with tabs.

Signed-off-by: Davor Joja <davorjoja@logicbricks.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
---
 drivers/gpu/drm/xylon/xylon_logicvc.h    |   16 -----------
 drivers/gpu/drm/xylon/xylon_logicvc_hw.h |   42 ++++++++++++++++++++---------
 2 files changed, 29 insertions(+), 29 deletions(-)

diff --git a/drivers/gpu/drm/xylon/xylon_logicvc.h b/drivers/gpu/drm/xylon/xylon_logicvc.h
index c4e9058..60ee8a4 100644
--- a/drivers/gpu/drm/xylon/xylon_logicvc.h
+++ b/drivers/gpu/drm/xylon/xylon_logicvc.h
@@ -50,20 +50,4 @@ void xylon_cvc_layer_ctrl(struct xylon_cvc *cvc, int id, int op);
 
 void xylon_cvc_set_hw_color(struct xylon_cvc *cvc, int id, u32 color);
 
-void xylon_cvc_int_state(struct xylon_cvc *cvc, unsigned int type,
-			 bool enabled);
-u32 xylon_cvc_int_get_active(struct xylon_cvc *cvc);
-void xylon_cvc_int_clear_active(struct xylon_cvc *cvc, u32 active);
-void xylon_cvc_int_hw_enable(struct xylon_cvc *cvc);
-void xylon_cvc_int_hw_disable(struct xylon_cvc *cvc);
-int xylon_cvc_int_request(struct xylon_cvc *cvc, unsigned long flags,
-			  irq_handler_t handler, void *dev);
-void xylon_cvc_int_free(struct xylon_cvc *cvc, void *dev);
-
-void xylon_cvc_reset(struct xylon_cvc *cvc);
-void xylon_cvc_enable(struct xylon_cvc *cvc, struct videomode *vmode);
-void xylon_cvc_disable(struct xylon_cvc *cvc);
-
-struct xylon_cvc *xylon_cvc_probe(struct device *dev, struct device_node *node);
-
 #endif /* _XYLON_LOGICVC_H_ */
diff --git a/drivers/gpu/drm/xylon/xylon_logicvc_hw.h b/drivers/gpu/drm/xylon/xylon_logicvc_hw.h
index f49b7bf..97254a2 100644
--- a/drivers/gpu/drm/xylon/xylon_logicvc_hw.h
+++ b/drivers/gpu/drm/xylon/xylon_logicvc_hw.h
@@ -17,18 +17,34 @@
 #ifndef _XYLON_LOGICVC_HW_H_
 #define _XYLON_LOGICVC_HW_H_
 
-#define LOGICVC_INT_L0_UPDATED    (1 << 0)
-#define LOGICVC_INT_L1_UPDATED    (1 << 1)
-#define LOGICVC_INT_L2_UPDATED    (1 << 2)
-#define LOGICVC_INT_L3_UPDATED    (1 << 3)
-#define LOGICVC_INT_L4_UPDATED    (1 << 4)
-#define LOGICVC_INT_V_SYNC        (1 << 5)
-#define LOGICVC_INT_E_VIDEO_VALID (1 << 6)
-#define LOGICVC_INT_FIFO_UNDERRUN (1 << 7)
-#define LOGICVC_INT_L0_CLUT_SW    (1 << 8)
-#define LOGICVC_INT_L1_CLUT_SW    (1 << 9)
-#define LOGICVC_INT_L2_CLUT_SW    (1 << 10)
-#define LOGICVC_INT_L3_CLUT_SW    (1 << 11)
-#define LOGICVC_INT_L4_CLUT_SW    (1 << 12)
+#define LOGICVC_INT_L0_UPDATED		(1 << 0)
+#define LOGICVC_INT_L1_UPDATED		(1 << 1)
+#define LOGICVC_INT_L2_UPDATED		(1 << 2)
+#define LOGICVC_INT_L3_UPDATED		(1 << 3)
+#define LOGICVC_INT_L4_UPDATED		(1 << 4)
+#define LOGICVC_INT_V_SYNC		(1 << 5)
+#define LOGICVC_INT_E_VIDEO_VALID	(1 << 6)
+#define LOGICVC_INT_FIFO_UNDERRUN	(1 << 7)
+#define LOGICVC_INT_L0_CLUT_SW		(1 << 8)
+#define LOGICVC_INT_L1_CLUT_SW		(1 << 9)
+#define LOGICVC_INT_L2_CLUT_SW		(1 << 10)
+#define LOGICVC_INT_L3_CLUT_SW		(1 << 11)
+#define LOGICVC_INT_L4_CLUT_SW		(1 << 12)
+
+void xylon_cvc_int_state(struct xylon_cvc *cvc, unsigned int type,
+			 bool enabled);
+u32 xylon_cvc_int_get_active(struct xylon_cvc *cvc);
+void xylon_cvc_int_clear_active(struct xylon_cvc *cvc, u32 active);
+void xylon_cvc_int_hw_enable(struct xylon_cvc *cvc);
+void xylon_cvc_int_hw_disable(struct xylon_cvc *cvc);
+int xylon_cvc_int_request(struct xylon_cvc *cvc, unsigned long flags,
+			  irq_handler_t handler, void *dev);
+void xylon_cvc_int_free(struct xylon_cvc *cvc, void *dev);
+
+void xylon_cvc_reset(struct xylon_cvc *cvc);
+void xylon_cvc_enable(struct xylon_cvc *cvc, struct videomode *vmode);
+void xylon_cvc_disable(struct xylon_cvc *cvc);
+
+struct xylon_cvc *xylon_cvc_probe(struct device *dev, struct device_node *node);
 
 #endif /* _XYLON_LOGICVC_HW_H_ */
-- 
1.7.5.4

